config.toml

```diff
@@ -1,5 +1,5 @@
 # The URL the site will be built for
-base_url = "https://reswitched-weekly-reboot.github.io"
+base_url = "https://swiftloke.github.io/reswitched-weekly-reboot.github.io"
```

Comment:
Same problem Cherry pick the build commits out and probably submit the pull request from a different branch